### PR TITLE
Make the index page a bit more up to date

### PR DESCRIPTION
**Pull request by annashipman**
These are some small edits to bring the index page a bit more up to date as we are about to start pointing people at this site. There are various other changes that could be made but I think this is enough for now. @dcarley would you be able to review this please?

cc/ @jystewart

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,24 +15,22 @@
...
 
     <p>
-      <a href="http://alpha.gov.uk/">Alpha.gov.uk</a> was the proof of concept that demonstrated
-      that the GOV.UK project (and the Government Digital Service with it) could work.
-      It was launched in May 2011 and remained live until the beta of GOV.UK launched
-      at the end of January 2012. This GitHub account was set up for that project and
-      then became the main account for the Government Digital Service.
+      As well as the alphagov organisation, we use
+
+      <a href="https://github.com/gds-operations/">GDS operations</a> to store our <a href="https://gds-operations.github.io/">open source
```
Comment (rboulton) [inline index.html]: There's also https://github.com/openregister

Comment (annashipman) [inline index.html]: That's deliberately separate for the time being.

**Comment by annashipman** [comment]
NB: anyone else who wants to review and/or merge please feel free! Doesn't have to be Dan

**Comment by dcarley** [comment]
View locally with `bundle exec jekyll serve --watch`

**Pull request merged**